nix/scripts/node_modules.sh

```diff
@@ -61,6 +61,8 @@ findFilesNewerThan() {
       -not -ipath "*/*android/build/*" -prune \
       -not -ipath "*/xcuserdata/*" -prune \
       -not -ipath "*/scripts/.packager.env" \
+      -not -path "*/node_modules/react-native/ReactCommon/react/renderer/components/rncore/*" \
```

Comment:
Adding a comment on why we ignore this path will help in the future 

```suggestion
     # react-native generates these files on every `xcodebuild` with `codegen`
      -not -path "*/node_modules/react-native/ReactCommon/react/renderer/components/rncore/*" \
```